schema.sql

```diff
@@ -35,3 +35,26 @@ drop column species;
 alter table animals
 add column species_id integer references species(species_id),
 add column owner_id integer references owners(owner_id);
+
+
+/* Create table named 'Vets' */
+create table vets(
+    vet_id serial primary key,
+    vet_name varchar, 
```

Comment:
- [ ] Setting limits on VARCHAR columns in a database ensures data integrity, improves performance, and enhances security by preventing overly long or inappropriate data. Kindly add a limit to the name field 👍